### ISSUE TITLE
Kapow Issue 27-29/04

### ISSUE THREAD

**Issue by Robinmcgvr** [Robinmcgvr/Kapow-Data#241]
There have been some issues with data over the period 27-29th April
Matteo is still working on it
Sites affected are:
<html>
<body>
[27 to 29 April slow server Issue.xlsx](https://github.com/Robinmcgvr/Kapow-Data/files/15185609/27.to.29.April.slow.server.Issue.xlsx)

**Comment by Lynne12** [comment]
@Robinmcgvr could you put this in a more readable format, please?  I am loading this affected week at the moment.

**Comment by Robinmcgvr** [comment]
@Lynne12 
Just done it after seeing it came out weird,
Probably best you hold off on loading this week for the rest of this week

**Comment by Robinmcgvr** [comment]
[27 to 29 April slow server Issue.xlsx](https://github.com/Robinmcgvr/Kapow-Data/files/15185639/27.to.29.April.slow.server.Issue.xlsx)

**Comment by Robinmcgvr** [comment]
The data that is missing is shown in column D

**Comment by Lynne12** [comment]
Only 2 websites relate to my online scheduling loading, so I should be okay to load the rest.

**Comment by Robinmcgvr** [comment]
n.b those sites that show a week or so of missing data, work like this every month we always have a lag of 5 to 7 days on those

**Comment by Robinmcgvr** [comment]
Yes prob more of an issue for OPR than loading, however the server is still playing up a bit, so that may not be the end of it yet
